serverless/functions/serverless-developing-python-functions.adoc

```diff
@@ -0,0 +1,17 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-developing-python-functions"]
+= Developing Python functions
+:context: serverless-developing-python-functions
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+:FeatureName: {FunctionsProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+Before you can develop functions, you must complete the set up steps in xref:../../serverless/functions/serverless-functions-setup.adoc#serverless-functions-setup[Setting up {FunctionsProductShortName}].
+
+include::modules/serverless-python-template.adoc[leveloffset=+1]
+include::modules/serverless-invoking-python-functions.adoc[leveloffset=+1]
```

Comment:
I feel like I'm missing part of the story of this assembly. The modules about invoking and return values seem like they're part of a bigger process.